### DISCUSSION TITLE
Autoloader script in WP gutenberg custom html block - i want to change the text color, any ideas how?

### DISCUSSION THREAD

**Discussion by easyname889** [markmap/markmap#258] [Q&A]
This is probably very simple and is done in CSS somewhere but i cant figure out how i change the text color in the html code?

Or is it possible with the YAML instructions for the markmap to define text colors?

```
<style>
  .markmap {
    position: relative;
  }
  .markmap > svg {
    width: 100%;
    height: 80vh;
  }
</style>

<div class="markmap">
  <script type="text/template">
---
title: ThisIsNotATest
markmap:
  colorFreezeLevel: 3
  initialExpandLevel: 3
---

```

**Comment by gera2ld** [comment]
Does this work for you?

```html
<style>
svg.markmap {
  color: blue;
}
</style>
```

**Comment by easyname889** [comment]
Yeah that did the trick =) thanks!

Den fre 28 juni 2024 kl 11:12 skrev Gerald ***@***.***>:
